godtoolsTests/UnitTests/App/Share/Data/DeepLinkingService/DeepLinkingServiceTests.swift

```diff
@@ -37,9 +37,11 @@ class DeepLinkingServiceTests: XCTestCase {
         case legacyKnowgodTractTeachMeToShareEnglishPage4 = "https://knowgod.com/en/teachmetoshare/4"
         case legacyKnowgodTractTeachMeToShareEnglishWithLiveShareStream = "https://knowgod.com/en/teachmetoshare?icid=gtshare&primaryLanguage=en&liveShareStream=acd9bee66b6057476cee-1612666248"
         case legacyKnowgodTractTeachMeToShareSpanishAndRussian = "https://knowgod.com/es/teachmetoshare?primaryLanguage=es&parallelLanguage=ru"
+        case legacyKnowgodLessonListen = "https://knowgod.com/en/lessonlisten/2"
         case knowgodTractKpgEnglish = "https://knowgod.com/en/tool/v1/kgp"
         case knowgodTractKpgEnglishPage1 = "https://knowgod.com/en/tool/v1/kgp/1"
         case knowgodLessonLessonConvoEnglish = "https://knowgod.com/lessons/lessonconvo/en"
+        case knowgodLessonLessonListenEnglish = "https://knowgod.com/en/lesson/lessonlisten/7"
```

Comment:
Hey @levieggertcru, I didn't see a test for the existing production version of Know God deep links for lessons, so I added that here (`legacyKnowgodLessonListen`).  They're getting processed in the legacy KnowGod deep link parser, but as a `tract` rather than a `lesson`.  

I'm also not sure where the `knowgodLessonLessonConvoEnglish` patten comes from-- is it legitimate?  It doesn't match the existing pattern or the new one (new is `knowGodLessonLessonListenEnglish`), and in the test it gets handled in `LegacyKnowGodDeepLinkParser`, but as a lesson.  Felt sketchy but didn't want to touch anything without asking!

Reply:
Hey @rachaelblue I posted this in Slack and the legacy can be removed.  Don't worry about removing it for this.  I created a new Jira task to remove the legacy parsing.  Would rather have you getting to the personalized tools work.

Reply:
And thank you for adding that test that wasn't there!  Will be good to have that test for the existing production version.